data/_models/shared/Resources.csv

```diff
@@ -68,12 +73,12 @@ Resources,,contributors,heading,,,,,,,,,List of relevant collaborations,"DataCat
 Resources,,people involved,refback,,,,Contacts,,resource,,,Persons involved and their role(s),"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE"
 Resources,,contact point,ref,,,,Contacts,,,,dcat:contactPoint,This attribute is written during migration of data from staging areas and used to produce valid dcat output. Contact point for this resource,DataCatalogueFlat
 Resources,,organisations involved,refback,,,,Organisations,,resource,,,Organisations involved and their role(s),"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,UMCGCohortsStaging,NetworksStaging,UMCUCohorts,INTEGRATE"
-Resources,,publisher,ref,,,,Organisations,,,,dcterms:publisher,This attribute is written during migration of data from staging areas and used to produce valid dcat output. Organisation that published this resource,DataCatalogueFlat
+Resources,,publisher,ref_array,,,,Organisations,,,,dcterms:publisher,This attribute is written during migration of data from staging areas and used to produce valid dcat output. Organisation that published this resource,DataCatalogueFlat
 Resources,,creator,ref_array,,,,Organisations,,,,dcterms:creator,This attribute is read only and written during migration of data from staging areas and used to produce valid dcat output. Organisation(s) that created this resource,DataCatalogueFlat
 Resources,,networks involved,refback,,,,Resources,,resources,,,The consortia or networks that this resource is part of,"DataCatalogueFlat,EMA"
 Resources,,networks other,text,,,,,,,,,List the names of any other networks that are not listed and this resource is involved in,EMA
 Resources,,available data and samples,heading,,,,,,,,,Data model and contents,"DataCatalogueFlat,EMA,RWEStaging,CohortsStaging"
-Resources,,datasets,refback,,,,Datasets,,resource,,,List of datasets contained in this resouce if applicable,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging"
+Resources,,datasets,refback,,,,Datasets,,resource,,"dcat:DatasetSeries,dcat:Dataset,ldp:contains",List of datasets contained in this resouce if applicable,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging"
```

Comment:
are these not dataset distributions?
emx2:Datasets is afaik not a dcat:Dataset(Series)?

Reply:
And how about nested catalogues?

Reply:
we need a refback to Resources, which 
in the case of rdf:type=dcat:Catalog should be  dcat:Catalogue:dataset 
in case of rdf:type=dcat:DatasetSeries

Reply:
should be dcat:datasets?